sys/shell/cmds/vfs.c

```diff
@@ -720,6 +720,91 @@ static int _vfs_handler(int argc, char **argv)
 
 SHELL_COMMAND(vfs, "virtual file system operations", _vfs_handler);
 
+#if MODULE_SHELL_CMD_GENFILE
+static char _get_char(unsigned i)
+{
+    i %= 2 * (1 + 'z' - 'a') + 10;
```

Comment:
I think it would be more readable to write `i %= 62; /* a-z, A-Z, 0..9, -> 62 characters */` (or use `#define` or what ever floats your boat)